src/test/java/com/synopsys/integration/alert/performance/BasePerformanceTest.java

```diff
@@ -0,0 +1,183 @@
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.common.rest.model.MultiFieldModel;
+import com.synopsys.integration.alert.performance.model.BlackDuckPerformanceProperties;
+import com.synopsys.integration.alert.performance.utility.AlertRequestUtility;
+import com.synopsys.integration.alert.util.TestTags;
+import com.synopsys.integration.bdio.model.Forge;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import com.synopsys.integration.blackduck.api.generated.view.ProjectVersionComponentView;
+import com.synopsys.integration.blackduck.api.generated.view.ProjectVersionView;
+import com.synopsys.integration.blackduck.configuration.BlackDuckServerConfig;
+import com.synopsys.integration.blackduck.configuration.BlackDuckServerConfigBuilder;
+import com.synopsys.integration.blackduck.service.BlackDuckService;
+import com.synopsys.integration.blackduck.service.BlackDuckServicesFactory;
+import com.synopsys.integration.blackduck.service.dataservice.ProjectBomService;
+import com.synopsys.integration.blackduck.service.dataservice.ProjectService;
+import com.synopsys.integration.blackduck.service.model.ProjectVersionWrapper;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.log.IntLogger;
+import com.synopsys.integration.log.Slf4jIntLogger;
+
+@Tag(TestTags.DEFAULT_PERFORMANCE)
+public abstract class BasePerformanceTest {
+    public static final String ROLE_ALERT_ADMIN = "ALERT_ADMIN";
+    protected final IntLogger intLogger = new Slf4jIntLogger(LoggerFactory.getLogger(this.getClass()));
+
+    private final BlackDuckPerformanceProperties blackDuckProperties = new BlackDuckPerformanceProperties();
+    private final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
+
+    private final String localAlertURL = "http://localhost/";
+
+    private AlertRequestUtility alertRequestUtility;
+    private BlackDuckServicesFactory blackDuckServicesFactory = null;
+
+    public abstract AlertRequestUtility createAlertRequestUtility();
+
+    @BeforeEach
+    public void setupTests() {
+        alertRequestUtility = createAlertRequestUtility();
+    }
+
+    public void triggerBlackDuckNotification() throws Exception {
```

Comment:
Don't you think this would be better suited for a test utility as well rather than a superclass?